docs/csharp/programming-guide/classes-and-structs/using-properties.md

```diff
@@ -1,39 +1,41 @@
 ---
 title: "Using Properties - C# Programming Guide"
 description: These examples illustrate using properties in C#. See how the get and set accessors implement read and write access and find out about uses for properties.
-ms.date: 07/29/2022
+ms.date: 03/15/2024
 helpviewer_keywords: 
   - "set accessor [C#]"
   - "get accessor [C#]"
   - "properties [C#], about properties"
 ---
 # Using Properties (C# Programming Guide)
 
-Properties combine aspects of both fields and methods. To the user of an object, a property appears to be a field; accessing the property requires the same syntax. To the implementer of a class, a property is one or two code blocks, representing a [get](../../language-reference/keywords/get.md) accessor and/or a [set](../../language-reference/keywords/set.md) accessor. The code block for the `get` accessor is executed when the property is read; the code block for the `set` accessor is executed when the property is assigned a value. A property without a `set` accessor is considered read-only. A property without a `get` accessor is considered write-only. A property that has both accessors is read-write. You can use an `init` accessor instead of a `set` accessor to make the property read-only.
+Properties combine aspects of both fields and methods. To the user of an object, a property appears to be a field; accessing the property requires the same syntax. To the implementer of a class, a property is one or two code blocks, representing a [`get`](../../language-reference/keywords/get.md) accessor and/or a [`set`](../../language-reference/keywords/set.md), or [`init`](../../language-reference/keywords/init.md) accessor. The code block for the `get` accessor is executed when the property is read; the code block for the `set` or `init` accessor is executed when the property is assigned a value. A property without a `set` accessor is considered read-only. A property without a `get` accessor is considered write-only. A property that has both accessors is read-write. You can use an `init` accessor instead of a `set` accessor to make the property read-only but enable the property to be set as part of object initialization.
```

Comment:
```suggestion
Properties combine aspects of both fields and methods. To the user of an object, a property appears to be a field; accessing the property requires the same syntax. To the implementer of a class, a property is one or two code blocks, representing a [`get`](../../language-reference/keywords/get.md) accessor and/or a [`set`](../../language-reference/keywords/set.md) or [`init`](../../language-reference/keywords/init.md) accessor. The code block for the `get` accessor is executed when the property is read; the code block for the `set` or `init` accessor is executed when the property is assigned a value. A property without a `set` accessor is considered read-only. A property without a `get` accessor is considered write-only. A property that has both accessors is read-write. You can use an `init` accessor instead of a `set` accessor to enable the property to be set as part of object initialization but otherwise make it read-only.
```